.circleci/config.yml

```diff
@@ -129,7 +129,7 @@ executors:
     parameters:
       resource_class:
         type: string
-        default: large
+        default: xlarge
```

Comment:
I've been seeing some flakiness after rebasing on main the other day... Since I was already having some problems with tests, I didn't want to get false positives. I'll go back down to large and see what happens.